### ISSUE TITLE
:sparkles: feat : ViT 최종 모델 업로드

### ISSUE THREAD

**Issue by haeun1** [boostcampaitech6/level1-imageclassification-cv-08#29]
### 내용

- Data Splitting 개선 
- Augmentation 개선 
- Hyperparameters 조정
- Loss Function 선택
